charts/rstudio-connect/values.yaml

```diff
@@ -286,6 +286,14 @@ chronicleAgent:
     imagePullPolicy: IfNotPresent
   # -- If true, the chart will attempt to lookup the Chronicle Server address and version in the cluster
   autoDiscovery: true
+  # -- Defines resources for the posit-chronicle-agent container
+  resources: {}
+    # requests:
+    #   memory: "256Mi"
+    #   cpu: "100m"
+    # limits:
+    #   memory: "512Mi"
+    #   cpu: "200m"
```

Comment:
In the default case, are the agent and product sharing the same pool of memory? In other words, if the agent hit `OOM killed`, would that take down the colocated product as well?

Reply:
I'm not completely sure about this. As I understand it, the pod is sharing resources, but with no limits defined, it will consume whatever it needs from the node (unless/until the node runs out of memory).

I know Ian talked about the initContainer having a separate lifecycle from the main container. But I don't know if that applies to OOM situations as well.

@ianpittwood - any thoughts here?